### PR TITLE
Set white logo in dark themes programmatically

### PR DESCRIPTION
**Pull request by NWuensche**
As `?attr/welcome_logo` in `nav_header_main.xml/imageView` always
selects the default, black icon, we change the icon in the `onCreate()`
method for dark themes.

### CODE REVIEW

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java [hunk 1]
```diff
@@ -134,6 +135,14 @@ protected void onCreate(Bundle savedInstanceState) {
                 lastUpdateTimeView = (TextView)headerView.findViewById(R.id.lastUpdateTime);
             }
 
+            // Set white logo in the navigation bar in dark and dark (high contrast) theme
+            if (Themes.getCurrentTheme() != null && (Themes.getCurrentTheme() == Themes.Theme.DARK || Themes.getCurrentTheme() == Themes.Theme.DARK_CONTRAST)) {
```
Comment (tcitworld) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java]: Maybe deport this to a helper function inside Themes : `isCurrentThemeDark()` ?

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java [hunk 2]
```diff
@@ -134,6 +135,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                if(headerView != null) {
```
Comment (tcitworld) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java]: Merge both `if` conditions.

**Comment by NWuensche** [comment]
Both things are changed now.

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java [hunk 3]
```diff
@@ -229,6 +236,10 @@ private void updateTime() {
         EventBus.getDefault().register(this);
     }
 
+    private boolean isCurrentThemeDark() {
```
Comment (tcitworld) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java]: Sorry, I meant to have this into `fr.gaulupeau.apps.Poche.ui.Themes` as a public function instead.

**Comment by NWuensche** [comment]
I hope I did it right this time.

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java (hunk 3 above)
Comment (NWuensche) [reply app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java]: Thanks, I hope I've did it right this time.

**Pull request merged**